text/70_performance_testing.md

```diff
@@ -14,13 +14,13 @@ As mentioned above, defining performance for a particular system means determini
 
 A __service-oriented indicator__ measures how well a system is providing a service to a particular user.  These are measured from the user's perspective and quantify aspects of the system the user would directly care about.  Some examples of service-oriented indicators would be average page load time, response time, or what percentage of the time the system is available.
 
-There are two main subcategories of service-oriented indicators: availability, or how available is the system to the user.  This could mean anything from what percentage of the time can the system be accessed to how available different features are to the users at different times.  The second main category is response time - how long does it take for the system to respond to the user's input or return a result?
+There are two main subcategories of service-oriented indicators. The first is availability - how available is the system to the user?  This could mean anything from what percentage of the time can the system be accessed to how available different features are to the users at different times.  The second main category is response time - how long does it take for the system to respond to the user's input or return a result?
 
-An __efficiency-oriented indicator__ measures how efficiently a system makes use of available computational resources.  You can think of these as being measured "from the program's point of view", or at least from the computer's point of view.  Examples of efficiency-oriented indicators would be what percentage of CPU operations are being used for performing some functionality, how much disk space is being used, or how many events can concurrent users can the system handle on a particular server?  Although the last may seem like it may be a service-oriented indicator, the fact that it's from the system level - how many users can the system handle, rather than how does it appear to a specific user - means that it's an efficiency-oriented indicator.  Whenever you are measuring from the perspective of the system, as opposed to something _directly_ experienced by the user, you are testing an efficiency-oriented indicator.
```

Comment:
Changed the question mark at the end of "Examples of efficiency-oriented indicators would be ... or how many events can concurrent users can the system handle on a particular server?" to a period. Seems more natural to me because "how" is introducing clauses, not questions.